tests/Tests/EventStoreTest.php

```diff
@@ -20,9 +25,9 @@ class EventStoreTest extends TestCase
      */
     private $es;
 
-    protected function setUp()
+    protected function setUp(): void
     {
-        $uri = getenv('EVENTSTORE_URI') ?: 'http://127.0.0.1:2113';
+        $uri = getenv('EVENTSTORE_URI') ?: 'http://admin:changeit@127.0.0.1:2113';
```

Comment:
here is where I would extract credentials and pass to the underlying http client instead of having them around the code